src/app/core/ccd-browser-support/ccd-browser-support.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="browser-support" [class.screen-990]="isSolicitor" [class.hide]="!showUnsupportedBrowser">
```

Comment:
can we use a ngIf here instead of [class.hide] ? seems a bit simpler to me with ngIf. We don't need to define an extra class

Reply:
I have added class here to include additional styling in future such as animated transitions while hiding the band.